python/cudf/cudf/tests/test_rolling.py

```diff
@@ -35,17 +35,23 @@ def test_rollling_series_basic(data, index, agg, nulls, center):
     psr = pd.Series(data, index=index)
     gsr = cudf.Series(psr)
 
-    for window_size in range(1, len(data) + 1):
-        for min_periods in range(1, window_size + 1):
-            assert_eq(
-                getattr(
-                    psr.rolling(window_size, min_periods, center), agg
-                )().fillna(-1),
-                getattr(
-                    gsr.rolling(window_size, min_periods, center), agg
-                )().fillna(-1),
-                check_dtype=False,
-            )
+    try:
+        for window_size in range(1, len(data) + 1):
+            for min_periods in range(1, window_size + 1):
+                assert_eq(
+                    getattr(
+                        psr.rolling(window_size, min_periods, center), agg
+                    )().fillna(-1),
+                    getattr(
+                        gsr.rolling(window_size, min_periods, center), agg
+                    )().fillna(-1),
+                    check_dtype=False,
+                )
+    except AssertionError as e:
+        if agg == "count" and data != []:
+            pytest.xfail(reason="Differ from Pandas behavior for count")
+        else:
+            raise e
```

Comment:
Can we run the rolling operations outside the try / except to make sure we only catch the `assert_eq`?